code/game/objects/structures/stool_bed_chair_nest/bed.dm

```diff
@@ -530,29 +558,36 @@ GLOBAL_LIST_EMPTY(activated_medevac_stretchers)
 		details += "It's linked to a beacon located at: [get_area(linked_beacon)]. Coordinates: (X: [linked_beacon.x], Y: [linked_beacon.y]).</br>"
 
 	if(world.time < last_teleport)
-		details += "[span_warning("It's bluespace engine is currently recharging. The interface estimates: [round(last_teleport - world.time) * 0.1] seconds until i has recharged.")]</br>"
+		details += "[span_warning("It's bluespace engine is currently recharging. The interface estimates: [round(last_teleport - world.time) * 0.1] seconds until it has recharged.")]</br>"
 
 	. += span_notice("[details.Join(" ")]")
 
+/obj/item/roller/medevac/update_overlays()
+	. = ..()
+	var/display_timer_cooldown = CEILING((timer_cooldown) * 0.1, 1)
+	if(isturf(loc) || !display_timer_cooldown)
+		return
+	var/mutable_appearance/desc = mutable_appearance('icons/misc/12x12.dmi')
+	desc.maptext = MAPTEXT(display_timer_cooldown)
```

Comment:
Add an 's' to it so it's indicative of it being seconds and not it looking like a stack.